src/main/java/org/opentripplanner/middleware/auth/Auth0Users.java

```diff
@@ -121,6 +123,39 @@ public static User getUserByEmail(String email, boolean createIfNotExists) {
         return null;
     }
 
+    /**
+     * Method to trigger an Auth0 job to resend a verification email. Returns an Auth0 {@link Job} which can be
+     * used to monitor the progress of the job (using job ID). Typically the verification email goes out pretty quickly
+     * so there shouldn't be too much of a need to monitor the result.
+     */
+    public static Job resendVerificationEmail(String userId) {
+        try {
+            return getManagementAPI()
+                .jobs()
+                // FIXME: This may need to be the otp-admin client_id instead.
```

Comment:
Maybe? An OtpUser who is self-creating an account could have a need to request another verification email.

Reply:
I think the client_id is mainly used to tailor the contents of the verification email (i.e., it will include the application name/logo based on the provided client_id). So ultimately I think we do want to add this, but it's a bit more configuration that I wanted to defer on until we were sure.